includes/admin/class-pinterest-for-woocommerce-admin.php

```diff
@@ -491,7 +493,6 @@ public function maybe_redirect_to_middleware() {
 
 			if ( ! isset( $_GET[ PINTEREST_FOR_WOOCOMMERCE_PREFIX . '_nonce' ] ) || ! wp_verify_nonce( sanitize_key( $_GET[ PINTEREST_FOR_WOOCOMMERCE_PREFIX . '_nonce' ] ), 'go_to_middleware_url' ) || ! current_user_can( 'manage_woocommerce' ) ) {
 				wp_die( esc_html__( 'Cheatin&#8217; huh?', 'pinterest-for-woocommerce' ) );
-				return;
```

Comment:
:question: Why do we need that change?

Reply:
`wp_die` literally kills the execution of PHP, so that return is redundant here. And it shows a warning in the editor.

Reply:
Since it's outside scope, rolled back here https://github.com/woocommerce/pinterest-for-woocommerce/pull/299/commits/d29c1e9b2ddef8fa2dde8487d0aa4f6cc4803d04

Reply:
TIL, thanks. I'm fine either way.